qemu/tests/qemu_guest_agent.py

```diff
@@ -894,6 +894,31 @@ def gagent_check_thaw_unfrozen(self, test, params, env):
             test.fail("The return value of thawing an unfrozen fs is %s,"
                       "it should be zero" % ret)
 
+    @error_context.context_aware
+    def gagent_check_freeze_frozen(self, test, params, env):
+        """
+        Freeze the frozen fs
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        self.gagent.fsfreeze()
```

Comment:
Better to check gagent fsfreeze status first
if it's frozen:
 continue. 
if it's thawed:
run self.gagent.fsfreeze()

Reply:
Hi, xiagao, I think no need here. Because we will boot a new guest before running every case, and have cleared env after last case. FS frozen situation shouldn't happen at start. Even if it happens, we should report error and stop the case like now, shouldn't continue.